testdata/cgo/main.go

```diff
@@ -160,6 +160,9 @@ func main() {
 	println("len(C.GoStringN(nil, 0)):", len(C.GoStringN(nil, 0)))
 	println("len(C.GoBytes(nil, 0)):", len(C.GoBytes(nil, 0)))
 
+	// C.CBytes will return a random pointer every time, so this style test won't work for it.
+	/*println("C.CBytes(nil):", C.CBytes(nil))*/
```

Comment:
You can test whether the bytes are there after the call, and/or whether it roundtrips with `C.GoBytes`.